vmdb/app/controllers/ops_controller/settings/cap_and_u.rb

```diff
@@ -392,7 +392,7 @@ def build_ds_tree(storages)
       ds_node[:key] = "Datastore_" + s[:id].to_s
       name = "<b>#{s[:name]}</b> [#{s[:location]}]"
```

Comment:
@romanblanco wasn't there a way to retain <b> tags, idea was to show datastore name in bold text, this used to work in dhtmlxtrees.